app/services/get-overview.js

```diff
@@ -25,16 +25,28 @@ module.exports = function (id, organisationLevel) {
 }
 
 var calculateValues = function (results) {
+  var totals = { name: 'Total/Average', totalPercentage: 0, totalPoints: 0, totalContractedHours: 0, totalReduction: 0, totalTotalCases: 0 }
+  var thingToReturn = {}
```

Comment:
Maybe rename to `totalsToReturn` or something a bit more descriptive?

Reply:
Yea.....was kinda....yea, I don't even have an excuse 😆